src/main/java/nl/tudelft/ewi/devhub/server/web/resources/repository/AbstractProjectResource.java

```diff
@@ -600,29 +604,56 @@ public Response showSettings(@Context HttpServletRequest request) throws IOExcep
 	@POST
 	@Path("/branches/delete")
 	@Transactional
-	public Response deleteBehindBranch(@Context HttpServletRequest request,
-									   @FormParam("branchDeleteName") String branchDeleteName,
-									   @QueryParam("fatal") String fatal)
+	public Response deleteBranch(@Context HttpServletRequest request,
+                                 @FormParam("branchDeleteName") String branchDeleteName,
```

Comment:
Hmm okay will do this then. I changed it to an Enum. I'll see if I can add some clear documentation to it. The code size is already smaller.